### DISCUSSION TITLE
Uso de funciones de la libreria estandar en el TP3 - Cadenas Seguras

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#91] [Preguntas y respuestas] [answered]
Hola! cuando en la consigna se especifica que "Las funciones a implementar no pueden usar las funciones de la librería.", se refiere a funciones como 'strlen' no?

**Comment by martinvilu** [comment, marked as answer]
Exactamente.
